### DISCUSSION TITLE
[Regression] Windows install is crashing on Windows ARM (previously worked before last windows update)

### DISCUSSION THREAD

**Discussion by shleeable** [rustdesk/rustdesk#7517] [Feature Request]
### Bug Description

I've been using Windows ARM on a M1 Mac under VMWare.. and running rustdeck under the X64 emulation without issue until this week.

I've updated windows and now the Rustdeck client crashes.

### How to Reproduce

Try to connect to destination
VM crashes
reboot crashed VM..
Try to connect to destination
VM crashes

### Expected Behavior

connects as usual.

### Operating system(s) on local side and remote side

Windows 11 ARM

### RustDesk Version(s) on local side and remote side

1.2.3 -> 1.2.3

### Screenshots

NA

### Additional Context

There isn't a native Windows ARM build... could I donate towards getting this done?

**Comment by rustdesk** [comment]
It is not a small donation which can afford this feature. :(

**Reply by Coflys** [reply]
我也需要  windowns arm64 版本
